src/tests/helpers.ts

```diff
@@ -103,21 +104,23 @@ export function mockAPICalls(testName: string, url = MainnetUrl) {
 
 /**
  * @internal
- * returns either a standard mainnet infura url
+ * returns either a standard mainnet linkpool url
+ * @see https://docs.linkpool.io/docs/rpc_main
  * or the one with attached INFURA SECRET key from
  * UNSTOPPABLE_RESOLUTION_INFURA_PROJECTID env variable if any
  */
-export function secretInfuraLink(infuraProtocol: InfuraProtocol = InfuraProtocol.http): string {
+export function secretInfuraLink(providerProtocol: ProviderProtocol = ProviderProtocol.http): string {
```

Comment:
It is weird to return `https://main-rpc.linkpool.io` from a function with name `secretInfuraLink`. Please either name the function appropriately or move this piece somewhere.